quilt-cli/quilt_cli/__main__.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+# Copyright (C) Seqera Labs S.L.- All Rights Reserved
+# Unauthorized copying of this file, via any medium is strictly prohibited
+# Proprietary and confidential
+# Written by Harshil Patel <harshil.patel@seqera.io>
+#            Ben Sherman <ben.sherman@seqera.io>
+#            Graham Wright <graham.wright@seqera.io>
```

Comment:
The usual Apache 2.0 license boilerplate should be used

Reply:
Done